test/src/e2e_vm_tests/test_programs/should_pass/require_contract_deployment/call_increment_contract/src/main.sw

```diff
@@ -11,10 +13,15 @@ fn main() {
         gas: 10000
     }
     (5);
-    let result = abi.increment {
+    abi.increment {
         gas: 10000
     }
     (5);
+    let result = abi.get {
+        gas: 10000
```

Comment:
@adlerjohn this is still an issue. If I remove `gas: 10000` here, the test fails with `OutOfGas`. I'm opening an issue about that.